rosbag2_compression/include/rosbag2_compression/compression_options.hpp

```diff
@@ -12,21 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef ROSBAG2_CPP__COMPRESSION_OPTIONS_HPP_
-#define ROSBAG2_CPP__COMPRESSION_OPTIONS_HPP_
+#ifndef ROSBAG2_COMPRESSION__COMPRESSION_OPTIONS_HPP_
+#define ROSBAG2_COMPRESSION__COMPRESSION_OPTIONS_HPP_
 
 #include <string>
 
 #include "visibility_control.hpp"
 
-namespace rosbag2_cpp
+namespace rosbag2_compression
 {
 
 /**
  * Modes are used to specify whether to compress by individual serialized bag messages or by file.
  * rosbag2_cpp defaults to NONE.
  */
-enum class ROSBAG2_CPP_PUBLIC CompressionMode : uint32_t
+enum class ROSBAG2_COMPRESSION_PUBLIC CompressionMode: uint32_t
 {
   NONE = 0,
   FILE,
```

Comment:
This change was done in a recently merged PR. 
Can you rebase on master so it doesn't get reviewed again?